docs/designers-developers/developers/tutorials/create-block/wp-plugin.md

```diff
@@ -1,32 +1,31 @@
 # WordPress Plugin
 
-A new block in the WordPress block editor is added by creating a WordPress plugin, installing, and activating the plugin. This page covers the details of a WordPress plugin if you are not already familiar.
+A block is added to the block editor using a WordPress plugin. You can create your own plugin, and after installing and activating the plugin use the block. Let's first look at what makes up a WordPress plugin.
 
 ## Plugin Details
 
-A WordPress plugin is a set of files in a directory within the site's `wp-content/plugins` directory. For this example, we will use the `create-block` package to generate the necessary plugin files.
+A WordPress plugin is a set of files within the site's `wp-content/plugins` directory. For our tutorial, we will use the `create-block` package to generate the necessary plugin files.
 
 ### Switch to Working Directory
 
 (1A) If you do not plan to use `wp-env` change to your local WordPress plugin directory. For example in Local it is: `~\Local Sites\mywp\wp-content\plugins`
 
 -or-
 
-(1B) If you do use `wp-env` start, you can start from any directory for your project. `wp-env` will use it as a plugin directory for your site.
+(1B) If using `wp-env`, you can start from any directory for your project; `wp-env` will use map it as a plugin directory for your site.
 
 ### Generate Plugin Files
 
-(2) Once in the right directory for your environment, the next step is to
-Regardless of environment, run the Run the following command to generate plugin files:
+(2) Once in the right directory for your environment, the next step is to run the following command to generate plugin files:
 
-```
+```sh
 npx @wordpress/create-block gutenpride
```

Comment:
This is confusing, since we are telling them to use the `@wordpress/create-block` package but then the command is from a different package. I think using it as create-block everywhere makes more sense.

Reply:
It’s the same package used by npm 😃 It’s a bit confusing so it’s fine to use npx if you prefer. I will open PR to promote both in create block.

Reply:
👍